docs/rest_api.md

```diff
@@ -353,6 +353,8 @@ see the documentation of the OBM driver in question (read `docs/obm-drivers.md`)
 Request Body:
 	{"obm": { "type": <obm-subtype>,
 		<additional sub-type specific values>}
+	 "metadata": {"label_1": "value_1",
```

Comment:
i was using label since this was originally for cryptographic keys and I didn't want to overload the term 'key'